de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/AuthTestResourceTest.java

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.resources;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Base64;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import org.junit.Test;
+
+import de.dlr.sc.virsat.server.test.AGitAndJettyServerTest;
+
+public class AuthTestResourceTest extends AGitAndJettyServerTest {
+	
+	private static final String BASIC_SCHEME = "Basic";
+	
+	@Test
+	public void testAuthentication() {

```

Comment:
This class could use some refactoring to eliminate duplicated code.
If you are interested in the accessibility, you can just check response status instead of the whole response.

Reply:
I think we want to remove this resource after we added authentication to the real resources. So I would improve the new testcases then.